src/Epsagon.Dotnet.Instrumentation/ElasticSearch/ElasticSearchEventsHandler.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Elasticsearch.Net;
+using Epsagon.Dotnet.Core;
+using OpenTracing.Tag;
+using OpenTracing.Util;
+
+namespace Epsagon.Dotnet.Instrumentation.ElasticSearch
+{
+    public static class ElasticSearchEventsHandler
+    {
+        public static void HandleRequestCompleted(IApiCallDetails details)
+        {
+            using (var scope = GlobalTracer.Instance.BuildSpan(details.HttpMethod.ToString()).StartActive(finishSpanOnDispose: true))
```

Comment:
We should create the span when the request is created, and close it here (otherwise the duration will be wrong)